### DISCUSSION TITLE
How can I add games or install them to already existing prefixes?

### DISCUSSION THREAD

**Discussion by ghost** [PhoenicisOrg/phoenicis#2349] [Q&A]
If I wanna for example add a steam game mod launcher to phoenicis,then I want to select the existing prefix.

**Comment by plata** [comment]
Ok, now I get you're point of specifying the prefix in the wizard.

But: why don't you just run the mod launcher from Steam directly?

**Reply by ghost** [reply]
> why don't you just run the mod launcher from Steam directly?

What do you mean?
If I just open the game,then it doesn't open the mod launcher.

But that was an example,cause I also have non steam games with mod launchers.

**Reply by plata** [reply]
The best way would be to add a new method `prefix()` (similar to `name()`) to the [QuickScript](https://github.com/PhoenicisOrg/scripts/blob/master/Engines/Wine/QuickScript/Quick%20Script/script.js). To be used [here](https://github.com/PhoenicisOrg/scripts/blob/6067b5b3f32da5ba7e73db983d5892c90850fba1/Engines/Wine/QuickScript/Installer%20Script/script.js#L92).
Then the installer for the mod launcher could set the prefix.

**Reply by ghost** [reply]
What do you mean with that?
Can you give me an example?

**Reply by plata** [reply]
It could look like:
```Javascript
const SteamScript = include("engines.wine.quick_script.steam_script");

new SteamScript()
    .name("My Addon")
    .editor("someone")
    .author("Toadfield")
    .appId(123456)
    .prefix("my_game");
```
